tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkEvpTest.cs

```diff
@@ -164,6 +164,8 @@ protected virtual void CheckRuntimeValues(MockCIVisibilityTest targetTest)
         AssertTargetSpanExists(targetTest, CommonTags.OSArchitecture);
         AssertTargetSpanExists(targetTest, CommonTags.OSPlatform);
         AssertTargetSpanEqual(targetTest, CommonTags.OSVersion, CIVisibility.GetOperatingSystemVersion());
+        targetTest.Metrics[CommonTags.LogicalCpuCount].Should().Be(Environment.ProcessorCount);
+        targetTest.Metrics.Remove(CommonTags.LogicalCpuCount);
```

Comment:
It seems this `.Remove` is done in most of these tests is the `targetTest` reused? (just curious)

Reply:
is not reused nope. the test removes all the expected tags and then we assert that we no longer has any other tag in the test event.

Reply:
Ahh makes sense thanks!